doc/release/1.4.0-notes.rst

```diff
@@ -76,6 +76,13 @@ Deprecated features
 Backwards incompatible changes
 ==============================
 
+`scipy` changes
+---------------
+Support for NumPy functions exposed via the root SciPy namespace is deprecated
```

Comment:
This should be in a "Deprecated features" section.  When the names are actually removed, it is noted in that release under the "Backwards incompatiable changes" section.